packages/wdio-selenium-standalone-service/selenium-standalone-service.d.ts

```diff
@@ -8,3 +8,7 @@ declare namespace WebdriverIO {
         skipSeleniumInstall?: boolean;
     }
 }
+
+declare module "@wdio/selenium-standalone-service" {
+    export default WebdriverIO
```

Comment:
This export does not seem to work for me. 
My typescript says: Exports and export assignments are not permitted in module augmentations.

Reply:
That's fine, we using same approach everywhere, for example please see typing of allure reporter

Reply:
Do you have this error in wdio project or in your test project? 
If in wdio, it's ok, otherwise that's bad

Reply:
fixing...

Reply:
the problem is because selenium-standalone itself has some mess in typings